package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "plugins",
+  "name": "trunk-plugins",
```

Comment:
Can you back this file change out? Was there something you ran into that brought this up?

Reply:
nothing specific, must have happened since I renamed my fork as `trunk-plugins`